components/Form/types.ts

```diff
@@ -57,6 +57,7 @@ export interface GenericField {
   isMulti?: boolean;
   isMultiInit?: boolean;
   isMultiTrigger?: string;
+  buttonStyle?: string;
```

Comment:
Why are we adding a `buttonStyle` prop?

Reply:
this is to handle a different CSS style for this button as per users request
the button is created dynamically by a component used in various places throughout the app
the prop will pass the CSS class itself (thanks @mapleleafjack), if not the usual class will be applied